pyproject.toml

```diff
@@ -1,12 +1,173 @@
+[tool.cibuildwheel]
+build = [
+    "cp37-*",
+    "cp38-*",
+    "cp39-*",
+    "cp310-*",
+]
+
+    [tool.cibuildwheel.environment]
+    PIP_PREFER_BINARY = true
+
+    [tool.cibuildwheel.windows]
+    archs = [
+        "AMD64",
+    ]
+
+
+    [tool.cibuildwheel.macos]
+    archs = [
+        "x86_64",
+
+        # Unfortunately we don't have Apple Silicon support yet
+        # TODO: Fix this
+        # Current problem seems to be that installed libfftw3 does not provide arm64
+        # symbols
+        # "universal2",
+        # "arm64",
```

Comment:
Even though we don't provide this here at the moment, we can likely provide arm64 support via conda-forge.

Reply:
I am not at all familiar with building conda packages - is there anything that I need to do for conda support?

Reply:
No, nothing specifically. I can try running the recipe later with the changes proposed here just to make sure it still works. I just wanted to double check that the changes to `setup.py` are not going to break the builds there.

Reply:
Perfect, thank you. I wouldn't think the `setup.py` changes would be breaking, but would be good to check.